src/plugins/theme/aspectRatio.js

```diff
@@ -3,10 +3,11 @@
  */
 module.exports = function () {
   return {
-    'none': 0,
-    'square': [1, 1],
-    DEFAULT: [16, 9], // Popular photo aspect ratio
-    '4x3': [4, 3],
-    '2x1': [2, 1], // Our usual crop for cards
+    1: '1',
+    2: '2',
+    3: '3',
+    4: '4',
+    9: '9',
+    16: '16',
   }
```

Comment:
Because of the aspect ratio plugin update, this has been redone to work with the new plugin

Reply:
https://github.com/tailwindlabs/tailwindcss-aspect-ratio